tools/nodeset_compiler/nodeset_compiler.py

```diff
@@ -114,12 +114,6 @@
 nsCount = 0
 loadedFiles = list()
 
-# Remove duplicate entries in typesArray, which occur when a nodeset has more than one depend.
-tmp_list = []
```

Comment:
See my comment in #5471.
Let's deal with this in a separate PR. So we can merge the first batch fast.

Reply:
reverted this file back